tests/include/detray/test/cpu/navigation_validation.hpp

```diff
@@ -137,8 +137,9 @@ class navigation_validation : public test::fixture_base<> {
             const auto &start = truth_trace.front();
             const auto &track = start.track_param;
             trajectory_type test_traj = get_parametrized_trajectory(track);
-            min_pT = std::min(min_pT, track.pT());
-            max_pT = std::max(max_pT, track.pT());
+
+            min_pT = std::min(min_pT, math::fabs(track.pT(-1.f)));
```

Comment:
Same here: If the charge can be changed in the detector scan, it must be the same charge in the navigation validation

Reply:
Same here